src/Ups.php

```diff
@@ -114,7 +119,7 @@ public function getLogger()
      *
      * @return string
```

Comment:
You can omit all redundant docblocks then

Reply:
Done.